content/docs/migration_1.0/Python.md

```diff
@@ -41,6 +41,30 @@ with zenoh.open(zenoh.Config()) as session:
     # and it will undeclare the subscriber
 ```
 
+## Drop-callback has to be wrapped in `handlers.Callback`
+
+In the previous 0.11.0 version, it was possible to pass a drop-callback with the main callback in a tuple for operations like `Session.declare_subscriber`. However, it was also possible to pass a tuple with a "receiver" (renamed "handler" in 1.0.0) as second member, and that could lead to confuse for users.
```

Comment:
```suggestion
In the previous 0.11.0 version, it was possible to pass a drop-callback with the main callback in a tuple for operations like `Session.declare_subscriber`. However, it was also possible to pass a tuple with a "receiver" (renamed "handler" in 1.0.0) as second member, and that could lead to confusion for users.
```

Reply:
> and that could confuse users.

The audience is the user and so the user should be addressed directly.

Reply:
My suggestion was follow-up to Luca's suggestion which wasn't applied correctly in the previous commit.